webclient/static/scss/_app-layout.scss

```diff
@@ -24,12 +23,29 @@ body {
       }
     }
   }
+
+  &.app-oznameni {
+    &:not(.app-edit) {
+
+      #app-page-wrapper {
+        min-width: auto;
+
+        .container-fluid {
+          max-width: $container-max-width-xl;
+        }
+      }
+    }
+  }
 }
 
 .maintenance {
   font-size: 20px;
   background-color: #322348;
 
+  p {
+    margin-bottom: 0px !important;
```

Comment:
@jnihnat Nerozhodí toto odstavce i jinde, když to není vztažené jen k třídě s oznámením odstávky?

Reply:
Je to jen pod class maintenance, takze ne.

Reply:
Vida, přehlédl jsem, že tam není zavřená závorka. Tedy za mne ok.